ivy/functional/frontends/torch/tensor.py

```diff
@@ -1986,6 +1986,19 @@ def unique_consecutive(self, return_inverse, return_counts, dim):
     def cummax(self, dim):
         return torch_frontend.cummax(self, dim)
 
+    @with_unsupported_dtypes(
+        {
+            "2.0.1 and below": (
+                "float16",
+                "bfloat16",
```

Comment:
We seem to be having a bunch of unsupported dtype errors in the CI, could you please look into the same and add to the list here and request review once tests are passing? Thanks for contributing!